readme.md

```diff
@@ -21,8 +21,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://YaroslavHor.github.io/layout_stars/)
+- [TEST REPORT LINK](https://YaroslavHor.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description
<img width="945" alt="image" src="https://github.com/mate-academy/layout_stars/assets/64750958/b06ae37f-e4fb-4fe5-9076-b54226e2a165">